### PR TITLE
Added configmaps to deployment descriptors

### PR DESCRIPTION
**Pull request by bibryam**

### CODE REVIEW

File: environments/dev/configmap/app.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+username=dev
```
Comment (cloudbotinmarsat) [inline environments/dev/configmap/app.properties]: INFO Remove this hard-coded username. [rule](http://ec2-54-194-136-253.eu-west-1.compute.amazonaws.com:9000/coding_rules#rule_key=jproperties%3AS2068)

File: environments/test/configmap/app.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+username=test
```
Comment (cloudbotinmarsat) [inline environments/test/configmap/app.properties]: INFO Remove this hard-coded username. [rule](http://ec2-54-194-136-253.eu-west-1.compute.amazonaws.com:9000/coding_rules#rule_key=jproperties%3AS2068)

**Comment by cloudbotinmarsat** [comment]
SonarQube analysis reported 2 issues
* INFO 2 info

Watch the comments in this conversation to review them.

**Comment by hammerdj1** [comment]
test this please

**Pull request merged**